common/src/main/java/us/ilite/common/Data.java

```diff
@@ -165,6 +151,23 @@ public void closeWriters() {
         }
     }
 
+    public void checkWriters() {
```

Comment:
Please name this something that describes what the method actually does - it's better if a method name is verbose and descriptive rather than short and unclear.